packages/assets-controllers/src/TokensController.test.ts

```diff
@@ -30,50 +30,36 @@ const stubCreateEthers = (ctrl: TokensController, res: boolean) => {
 describe('TokensController', () => {
   let tokensController: TokensController;
   let preferences: PreferencesController;
-  let network: NetworkController;
-  let messenger: NetworkControllerMessenger;
 
-  let instEthProvStub: sinon.SinonStub;
+  let onNetworkStateChangeListener: (state: NetworkState) => void;
+  const changeNetwork = (providerConfig: ProviderConfig) => {
+    onNetworkStateChangeListener({
+      ...defaultNetworkState,
+      providerConfig,
+    });
+  };
+
+  const sepolia = { chainId: '11155111', type: 'sepolia' as NetworkType };
```

Comment:
Instead of using `as NetworkType` here what do you think about using `as const`? The former is a typecast, while the latter is a type inference.